docs/setup.rst

```diff
@@ -25,6 +26,17 @@ itemadapter.ItemAdapter.ADAPTER_CLASSES_ as early as possible in your code::
     ItemAdapter.ADAPTER_CLASSES.appendleft(ZyteItemAdapter)
 
 
+Alternatively, make your own subclass of :class:`itemadapter.ItemAdapter`::
+
+    from itemadapter import ItemAdapter
+    from zyte_common_items import ZyteItemAdapter
+
+    class MyItemAdapter(ItemAdapter):
+        ADAPTER_CLASSES = deque([ZyteItemAdapter]) + ItemAdapter.ADAPTER_CLASSES
```

Comment:
You also need a deque import in this example.

See also: https://github.com/scrapy/itemadapter/issues/71